src/vs/platform/terminal/node/ptyService.ts

```diff
@@ -275,7 +277,8 @@ export class PersistentTerminalProcess extends Disposable {
 		public readonly workspaceName: string,
 		public readonly shouldPersistTerminal: boolean,
 		cols: number, rows: number,
-		private readonly _logService: ILogService
+		private readonly _logService: ILogService,
+		private readonly _icon?: string
```

Comment:
I think we want this mandatory? At least `_icon: string | undefined`

Reply:
isn't the `?` doing the same thing as `string | undefined`?